posthog/hogql/grammar/HogQLParserVisitor.py

```diff
@@ -0,0 +1,548 @@
+# Generated from HogQLParser.g4 by ANTLR 4.11.1
+from antlr4 import *
+if __name__ is not None and "." in __name__:
+    from .HogQLParser import HogQLParser
+else:
+    from HogQLParser import HogQLParser
+
+# This class defines a complete generic visitor for a parse tree produced by HogQLParser.
```

Comment:
This is a sample generated visitor. I'm keeping it here so we could always track (via git diff) modified parse tree nodes if you run `pnpm run grammar:build`.